ui/components/component-library/label/label.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import classnames from 'classnames';
+import { Text, ValidTag } from '../text';
+import {
+  FontWeight,
+  TextVariant,
+  Display,
+  AlignItems,
+} from '../../../helpers/constants/design-system';
+
+export const Label: React.FC<{
+  htmlFor?: string;
+  className?: string;
+  children: string | React.ReactNode;
+}> = ({ htmlFor, className, children, ...props }) => (
```

Comment:
Would it be possible to move these to a `.types.ts` file and export the `LabelProps` that will match our other component conventions. Could we also forwardRef to the Text component. I think all of our components should forward refs

Reply:
Thanks for the suggestions, I made two commits with the changes